vickev/howdypix
# Add Album entity and fill the database.

## Description

Add Album entity and fill the database.





## How to test?

- Check-out this branch.
- Run `yarn dev`
- Go to `apps/server` and run `yarn typeorm query "SELECT * from Album"` => Should return the albums.

## Checklist
